tools/localization_analysis/include/localization_analysis/map_matcher.h

```diff
@@ -35,6 +35,9 @@ class MapMatcher {
   MapMatcher(const std::string& input_bag_name, const std::string& map_file, const std::string& image_topic,
              const std::string& output_bag_name, const std::string& config_prefix = "");
   void AddMapMatches();
+  int match_count;
```

Comment:
For our style guide, we normally add an underscore _ suffix to member variables. As another note, if you want some more stats for the  feature counts, I have an "Averager" object here: https://github.com/nasa/astrobee/blob/master/localization/localization_common/include/localization_common/averager.h
You can simply call Update(...) with the feature count, then when you want to print the stats to terminal call Log(), and you'll get the mean, min, max, stddev.